src/modules/verification/dto/verificationDto.ts

```diff
@@ -0,0 +1,18 @@
+import { IsNumber, IsPhoneNumber } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class VerificationDto {
```

Comment:
Can you rename this file to `.dto.ts`? 
I think there was some library expecting the file name convention.

The naming might also not be ideal. Maybe `PhoneNumberVerificationDto` is a bit easier to follow (especially in the swagger schemas).